src/Offer/Popularity/Popularity.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\Popularity;
+
+class Popularity
```

Comment:
Good idea to not extend the base classes we used to use. Would be good if we can remove those at some point in the long run 👍